src/components/select-field/__tests__/__snapshots__/BaseSelectField.test.js.snap

```diff
@@ -5,7 +5,7 @@ exports[`components/select-field/BaseSelectField renderSelectButton() should sen
   aria-activedescendant={null}
   aria-autocomplete="list"
   aria-expanded={false}
-  aria-owns="selectfield11"
+  aria-owns="selectfield12"
```

Comment:
Not sure if this change is needed. This happened after I used the injectIntl HOC on BaseSelectField component. Would like feedback on whether or not this is needed.

Reply:
This is an autoincremented counter for the generated IDs used to associate aria labels with their elements. It went up because a new test was added, so the uid generator was called again